Documentation/package-dependency-flow.md

```diff
@@ -43,32 +43,46 @@ ability to service the product.
 ## Package Versions
 
 The package versions referenced during a source build can be fixed or dynamic.
-Having a dependency in the Version.Details.xml file along with a corresponding 
-Versions.props version property signals to source build to lift the version to the
-latest. Let's look at an example to illustrate the behavior.
+Package versions will get lifted dynamically if the following conditions are met:
 
-**Versions.props**
+1. The dependency is declared in the Version.Details.xml file.
 
-```xml
-...
-  <SystemCommandLineVersion>2.0.0-beta4.24068.1</SystemCommandLineVersion>
-...
-```
+    **Version.Details.xml**
 
-**Version.Details.xml**
+    ```xml
+    ...
+      <Dependency Name="System.CommandLine" Version="2.0.0-beta4.24068.1">
+        <Uri>https://github.com/dotnet/command-line-api</Uri>
+        <Sha>02fe27cd6a9b001c8feb7938e6ef4b3799745759</Sha>
+      </Dependency>
+    ...
+    ```
 
-```xml
-...
-  <Dependency Name="System.CommandLine" Version="2.0.0-beta4.24068.1">
-    <Uri>https://github.com/dotnet/command-line-api</Uri>
-    <Sha>02fe27cd6a9b001c8feb7938e6ef4b3799745759</Sha>
-  </Dependency>
-...
-```
+1. A corresponding version property is defined in the Versions.props.
+
+    **Versions.props**
+
+    ```xml
+    ...
+      <SystemCommandLineVersion>2.0.0-beta4.24068.1</SystemCommandLineVersion>
```

Comment:
This property would also need to be referenced in order to have an effect.

Reply:
Added clarity in the opening and closing sections of the _Package Versions_ section.